core/services/ocr2/plugins/functions/integration_tests/v1/functions_integration_test.go

```diff
@@ -91,7 +64,7 @@ func TestIntegration_Functions_MultipleV1Requests_WithUpgrade(t *testing.T) {
 
 	utils.SetupRouterRoutes(t, b, owner, routerContract, active.Address, proposed.Address, allowListContractAddress)
 
-	_, _, oracleIdentities := utils.CreateFunctionsNodes(t, owner, b, routerAddress, nOracleNodes, maxGas, nil, nil)
+	_, _, oracleIdentities := utils.CreateFunctionsNodes(t, owner, b, routerAddress, nOracleNodes, maxGas, utils.ExportedOcr2Keystores, utils.MockThresholdKeyShares)
```

Comment:
Do we have to change it?

Reply:
Yes.  Else tests fail. However, we could modify this test to exclude secrets entirely.

Reply:
I decided I preferred to keep the threshold secrets here as it represents a more complete test of the system.  I also kept the first test which doesn't use any secrets

Reply:
It's fine. As long as we have a test without any secrets (the first one) then using them here doesn't matter that much.